gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -38,6 +39,12 @@ pub struct ClnExtensionOpts {
 // Note: Once this binary is stable, we should be able to remove current 'ln_gateway'
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
+    tracing_subscriber::fmt()
+        .with_env_filter(
+            EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info")),
+        )
+        .init();
+
```

Comment:
You can't use the default subscriber that prints to stdout in CLN plugins since they use stdin/out for communication with CLN. [`cln-pugin` brings its own subscriber which formats log events as valid CLN JSON messages. We should try to understand it better.](https://github.com/ElementsProject/lightning/blob/master/plugins/src/logging.rs)

Reply:
Just realized this when debugging changes in this PR!
From what @dpc has pointed out, the log level flag is controllable with `CLN_PLUGIN_LOG`

Validating. And closing this PR as no-fix

Reply:
`tracing_subscriber` defaults to stdout? Bad default.